### PR TITLE
Ingesting video annotations from Next Video Editor

### PR DESCRIPTION
**Pull request by tosan88**
Supporting ingesting video annotations from Next Video Editor into Neo4j.
New services:
* `next-video-content-annotator` - no changes on code repo
* `next-video-annotations-rw-neo4j` - see https://github.com/Financial-Times/annotations-rw-neo4j/pull/22

**Note**: I will update the release version of next-video-annotations-rw-neo4j before going to pre-prod. I submitted this PR because there were other changes as well, so I wanted to get feedback as early as possible.

See also https://sites.google.com/a/ft.com/universal-publishing/documentation/story-specs/3-mar-2017-next-video-platform-work for architecture diagram and other useful information

### CODE REVIEW

File: services.yaml [hunk 1]
```diff
@@ -303,6 +303,16 @@ services:
   version: v3.0.2
   count: 3
   sequentialDeployment: true
+- name: next-video-annotations-rw-neo4j-sidekick@.service
+  count: 2
+- name: next-video-annotations-rw-neo4j@.service
+  version: 0.2.0-vid-support-next-video-rc1
```
Comment (tamas-molnar) [inline services.yaml]: Deploying rc version to prod?

File: next-video-content-annotator@.service [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  --env "JAVA_OPTS=$JAVA_OPTS" \
+  --env "FILTER_NOVELTIES=true" \
+  --env "MESSAGE_FORWARDING_ENABLED=true" \
+  --env "FORWARD_TO_TOPIC_NAME=PostMetadataPublicationEvents" \
+  --env "KAFKA_PROXY=%H:8080" \
+  --env "KAFKA_PROXY_HOST_HEADER=kafka" \
+  --env "GRAPHITE_HOST=graphite.ft.com" \
+  --env "GRAPHITE_PORT=2003" \
+  --env "GRAPHITE_PREFIX=coco.services.$ENV.next-video-content-annotator.%i" \
+  up-registry.ft.com/coco/content-annotator:$DOCKER_APP_VERSION;'
```
Comment (tamas-molnar) [inline next-video-content-annotator@.service]: Why not extract the image name here too?

File: services.yaml (hunk 1 above)
Comment (tosan88) [inline services.yaml]: See my Note in description. I will change this before going to pre-prod

File: next-video-content-annotator@.service (hunk 2 above)
Comment (tosan88) [inline next-video-content-annotator@.service]: Right, will do that

Comment (tosan88) [inline next-video-content-annotator@.service]: Done

**Pull request merged**